libafl/src/observers/cmp.rs

```diff
@@ -725,14 +944,74 @@ impl Debug for AFLppCmpVals {
     }
 }
 
-/// The AFL++ `cmp_map` struct, use with `ForkserverCmpObserver`
+impl AFLppCmpVals {
+    #[must_use]
+    /// Reference comparison values as comparison operands
+    pub fn operands(&self) -> &[[AFLppCmpOperands; AFL_CMP_MAP_H]; AFL_CMP_MAP_W] {
+        unsafe { &self.operands }
+    }
+
+    #[must_use]
+    /// Mutably reference comparison values as comparison operands
+    pub fn operands_mut(&mut self) -> &[[AFLppCmpOperands; AFL_CMP_MAP_H]; AFL_CMP_MAP_W] {
```

Comment:
Shouldn't this return a `mut`? And why does it need unsafe?

Reply:
Yep it should return mut, fixing this. It needs unsafe because it's a `union`.